src/context/db.tsx

```diff
@@ -1,16 +1,30 @@
-import React, { createContext, useContext, useState } from "react";
+import React, {
+  createContext,
+  useContext,
+  useState,
+  FunctionComponent
+} from "react";
 import { RxDatabase } from "rxdb";
 
-export const DbContext = createContext({});
+interface DbContext {
+  db: RxDatabase;
+  setDb: Function;
+}
+
+export const DbContext = createContext<DbContext>({
+  db: undefined,
+  setDb: undefined
+});
 
-export const useDbContext = (): { db: RxDatabase; setDb: Function } =>
-  useContext(DbContext) as any;
+export const useDbContext = (): DbContext => useContext<DbContext>(DbContext);
 
-interface DbContextProviderProps {
+export interface DbContextProviderProps {
```

Comment:
Hmm do we still need this since we only use `children`?

Reply:
seemed like so